.github/check-wiki-changelog.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+source .github/bot-pr-base.sh
+
+echo "Checking whether the PR contains the 1:ST:skip-changelog-wiki-check label"
```

Comment:
I think it might be more descriptive if the label states its intention, not its effect: 
```suggestion
echo "Checking whether the PR contains the 1:ST:no-changelog-entry label"
```

Reply:
The label is intended for the action job, so I think it should say what the action job should do, which is to skip the check in the wiki whether the changelog has been added or not.

Reply:
To me the question is who is the primary intended audience for the label. The script doesn't really care what it is named, but we as developers can profit from a shorter label describing the PR's effect, not what we expect the Action to do.

Reply:
from the voting in our slack, we choose `1:ST:no-changelog-entry` label